docs/signature-layout.md

```diff
@@ -0,0 +1,29 @@
+# Signature File Layout
+
+A common file layout for storing and serving signatures provides a consistent way to reference image signatures. Signatures on a filesystem or a web server shall use this common layout. Signatures stored in a REST API are not required to use this common layout.
+
+## Specification
+
+**SCHEME://[URI]/PATH_PREFIX/REGISTRY/REPOSITORY/IMAGE@MANIFEST_DIGEST/signature-INT**
```

Comment:
Yeah, we don’t really care about what happens at the top level. There is a root of the hierarchy, and a structure of subdirectories/files below that root.